### PR TITLE
Change label formatting for skills, roles, materials

### PR DESCRIPTION
**Pull request by brein62**
Change UI formatting for skills, roles, and materials. Roles are placeholder red and materials placeholder green until the roles and materials count feature is implemented

Screenshot 2023-10-25 at 6 56 03 PM

### CODE REVIEW

File: src/main/java/seedu/address/ui/EventCard.java [hunk 1]
```diff
@@ -50,5 +56,11 @@ public EventCard(Event event, int displayedIndex) {
         dateAndTime.setText(event.getStartDate().toString() + " to " + event.getEndDate().toString());
         loc.setText("Location: " + event.getLocation().location);
         description.setText("Description: " + event.getDescription().description);
+        event.getRoles().stream()
+                .sorted(Comparator.comparing(role -> role.roleName))
+                .forEach(role -> roles.getChildren().add(new RoleLabel(role).getRoot()));
+        event.getMaterials().stream()
+                .sorted(Comparator.comparing(material -> material.material))
+                .forEach(material -> materials.getChildren().add(new MaterialLabel(material).getRoot()));
```
Comment (ruo-x) [inline src/main/java/seedu/address/ui/EventCard.java:64]: big brain

**Review by ruo-x — APPROVED** [review]
LGTM! very nice rly liked how you provided screenshots for us to visualize

File: src/main/java/seedu/address/ui/EventCard.java (hunk 1 above)
Comment (brein62) [reply src/main/java/seedu/address/ui/EventCard.java:64]: :D i love my mac

Comment (brein62) [reply src/main/java/seedu/address/ui/EventCard.java:64]: someone else did this i wasn't that smart :(

**Review by zacwong2151 — APPROVED** [review]
tbh im not v sure what im reading

**Comment by brein62** [comment]
> tbh im not v sure what im reading

hey :( hope the screenshot result help hehe

**Review by jordankanghm — APPROVED** [review]
sexy man

**Pull request merged**